lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems.</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+
+  - name: Replacing Extension
+    required: 'Yes'
+    type: Extension
+    description: The resource that this ChargeItem is replacing.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueId": "1",
+            "url": "item-version"
+          }
+        ],
+        "valueReference": {
+          "reference": "ChargeItem/317370800"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: performing-location
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: modify
+    example: |
+      {
+        "valueReference": {
+          "reference": "Location/21304903",
+          "display": "NM"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: description
+    required: 'No'
+    type: Extension
+    description: A description providing additional details of the resource.
+    action: modify
+    example: |
+      {
+        "valueString": "Acetaminophen Level",
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/description"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: custom-attribute
+    required: 'No'
+    type: Extension
+    description: A client defined custom attribute for the charge.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "id": "2572130093",
+            "valueString": "Special Authority",
+            "url": "custom-attribute-name"
+          },
+          {
+            "valueInteger": 1,
+            "url": "custom-attribute-value"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+      }
+    note: >
+      <ul>
+        <li>Custom-attribute-name and Custom-attribute-value are both required</li>
+        <li>Custom-attribute-value can be of type integer, string, or decimal</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: modifier-code
+    required: 'No'
+    type: Extension
+    description: A code providing information about the procedure performed on the patient associated to the resource.
```

Comment:
Should this be `...modification performed on the...` ? It's the same description as procedure-code, so I think we need to be specific about modifier-code vs procedure-code

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R286

Reply:
I agree it needs to change. Should we use the same description used for the Custom Extensions section for modifier-code? "A code providing additional detail about a product or service."

Reply:
Let's go with that description. The one I suggested ends up reading kinda strangely I realized haha.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/928dc349ae56b5c07470ad207306beef33bc2b8d#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R304